src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -0,0 +1,298 @@
+    int index = 0;
+
+    for (WebElement element : CdfSchemaLocators.outputSchemaColumnNames) {
+      actualOutputSchema.put(
+        ElementHelper.getElementAttribute(element, "value"),
+        ElementHelper.getElementAttribute(CdfSchemaLocators.outputSchemaDataTypes.get(index), "title"));
+      index++;
+    }
+
+    return actualOutputSchema;
+  }
+
+  /**
+   * Verify if the generated Output Schema matches the expected Schema
+   *
+   * @param schemaJsonArray Property key in the Plugin Properties file whose value is the Expected Schema
+   *                        ({@link ConstantsUtil#DEFAULT_PLUGIN_PROPERTIES_FILE})
+   */
+  public static void verifyOutputSchemaMatchesExpectedSchema(String schemaJsonArray) {
+    Map<String, String> expectedOutputSchema =
+      JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(schemaJsonArray));
+
+    Assert.assertTrue("Schema displayed on UI should match with the expected Schema",
+      getOutputSchema().equals(expectedOutputSchema));
+  }
+
+  /**
+   * Verify the Input Schema matches the Output Schema
+   *
+   * @param outputSchema Output Schema Field-Data type map
+   */
+  public static void verifyInputSchemaMatchesOutputSchema(Map<String, String> outputSchema) {
+    Map<String, String> actualInputSchema = new HashMap<>();
+    int index = 0;
+    for (WebElement element : CdfSchemaLocators.inputSchemaColumnNames) {
+      actualInputSchema.put(element.getAttribute("value"),
+        CdfSchemaLocators.inputSchemaDataTypes.get(index).getAttribute("title"));
+      index++;
+    }
+
+    Assert.assertTrue("Schema should match", actualInputSchema.equals(outputSchema));
+  }
+
+  /**
+   * Click on the Plugin's Properties page tab
+   *
+   * @param tabName Properties, Preview or Documentation
+   */
+  public static void clickOnTab(String tabName) {
+    ElementHelper.clickOnElement(CdfPluginPropertiesLocators.locateTab(tabName));
+  }
+
+  /**
+   * Verify 'Output Records' table columns in the Preview tab to check if it matches the Output Schema fields
+   *
+   * @param listOfFieldsInOutputSchema List of Output Schema fields
+   */
+  public static void verifyOutputRecordsTableColumnsUnderPreviewTabMatchesOutputSchemaFields
+  (List<String> listOfFieldsInOutputSchema) {
+    List<String> previewTableColumnList = new ArrayList<>();
+
+    for (WebElement element : CdfPluginPropertiesLocators.previewOutputRecordsTableColumnNames) {
+      previewTableColumnList.add(ElementHelper.getElementAttribute(element, "title"));
+    }
+
+    Assert.assertTrue("Schema column list should match the preview column list",
+      previewTableColumnList.equals(listOfFieldsInOutputSchema));
+  }
+
+  /**
+   * Verify 'Input Records' table columns in the Preview tab to check if it matches the Output Schema fields
+   *
+   * @param listOfFieldsInOutputSchema
+   */
+  public static void verifyInputRecordsTableColumnsUnderPreviewTabMatchesOutputSchemaFields
```

Comment:
input records column names in a plugin should match with the input schema instead of output schema

Reply:
Renamed the helper.